source/catalog.html

```diff
@@ -0,0 +1,351 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo-link" href="index.html">
+      <img class="main-header__logo-image" src="img/logo-header.svg" width="191" height="37" alt="логотип cat energy">
+    </a>
+    <nav class="main-header__navigation navigation-mobile">
+      <button class="navigation-mobile__toogle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <ul class="navigation-mobile__list">
+        <li class="navigation-mobile__item mobile-menu">
+          <a class="mobile-menu__title" href="/">Главная</a>
+        </li>
+        <li class="navigation-mobile__item mobile-menu">
+          <a class="mobile-menu__title" href="catalog.html">Каталог продукции</a>
+        </li>
+        <li class="navigation-mobile__item mobile-menu">
+          <a class="mobile-menu__title" href="form.html">Подбор программы</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="catalog-main">
+    <h1 class="catalog-main__main-title main-title">Каталог продукции</h1>
+    <section class="catalog-main__catalog catalog">
+      <h2 class="visually-hidden">Основные товары</h2>
+      <ul class="catalog__list">
+        <li class="catalog__item product">
+          <a class="product_link-title" href="product-page.html">
```

Comment:
У класса элемента должно быть два символа подчёркивания в качестве разделителя:
```suggestion
          <a class="product__link-title" href="product-page.html">
```
Часто забываете поставить второй символ. Все ошибки приводить не буду, просто проверьте [здесь](https://yoksel.github.io/html-tree/) (ссылку на этот сервис давал ранее) и устраните ошибки.